python/ccxt/async_support/base/exchange.py

```diff
@@ -38,6 +38,17 @@
 from ccxt.async_support.base.ws.future import Future
 from ccxt.async_support.base.ws.order_book import OrderBook, IndexedOrderBook, CountedOrderBook
 
+from ccxt.static_dependencies.aiohttp_socks import ProxyConnector
+
+# -----------------------------------------------------------------------------
+
```

Comment:
leftover?

Reply:
yes leftover, because I was not sure whether we should include it in static-deps or use it as optionally installed package (like we have curve eddsa package)